packages/sdk/src/conversationWithCloudAdapter/middlewares/cardActionMiddleware.ts

```diff
@@ -0,0 +1,106 @@
+import {
```

Comment:
Files under `conversationWithCloudAdapter/middlewares` and `conversationWithCLoudAdapter/sso` are similar with those under `conversation/` except comments, do we need to keep the copy of these files?

Reply:
Thanks, remove the duplicated files and redirect the references to the files under `conversation/`.